src/Nethermind/Nethermind.Serialization.Rlp/WithdrawalDecoder.cs

```diff
@@ -23,7 +23,7 @@ public class WithdrawalDecoder : IRlpStreamDecoder<Withdrawal>, IRlpValueDecoder
             Index = rlpStream.DecodeULong(),
             ValidatorIndex = rlpStream.DecodeULong(),
             Address = rlpStream.DecodeAddress(),
-            Amount = rlpStream.DecodeUInt256()
+            Amount = rlpStream.DecodeUlong()
```

Comment:
Networking rlp schema changes too, right?

Reply:
Isn't the numbers lower than 64 bytes decoded in the same way? We changed the type so of course we changed DecodeUlong to DecodeUInt256